app/services/importer/todoist.rb

```diff
@@ -0,0 +1,35 @@
+require 'open-uri'
+require 'json'
+
+class Importer::Todoist
+  API_URL = 'https://todoist.com/API/v7/sync'
+
+  def self.import(token)
+    request_url = create_url(token: token, sync_token: '*', resource_types: '["all"]')
+    todoist_tasks = api_request(request_url)
+    todoist_tasks["items"].map do |todoist_task| 
+      {
+        title: todoist_task["content"],
+        deadline_at: (todoist_task["due_date_utc"].presence || Time.zone.now.since(1.month)),
+        created_at: todoist_task["date_added"],
+        weight: 1
+      }
+    end
+  end
+
+  private
+
+  def self.api_request(request_url)
+    res = open(request_url)
+    code, message = res.status
+
+    if code == '200'
+      return JSON.parse(res.read)
+      end
```

Comment:
__[rubocop]__
* Lint/EndAlignment: `end` at 28, 6 is not aligned with `if` at 26, 4.